bin/watch-css.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -xe
+
+DIR="$( cd "$( dirname "$0" )" && pwd )"
+
+cd ${DIR}/../blueocean-web
+npm install ../blueocean-core-js
+npm install -g slink
```

Comment:
All the docs refer to slink as being used globally, but we could (and should) make sure it doesn't have to be